### PR TITLE
[Bigdl] Change all "bigdl" to "ipexllm"

### PR DESCRIPTION
**Pull request by yutianchen666**
Update bigdl-LLM to IPEX-LLM

### CODE REVIEW

File: docs/serve_ipex-llm.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Deploying and Serving LLMs with ipex-LLM
+[ipex-LLM](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/index.html) is a library for running LLM (large language model) on Intel XPU (from Laptop to GPU to Cloud) using INT4 with very low latency (for any PyTorch model).
```
Comment (xwu99) [inline docs/serve_ipex-llm.md:2]: Pls honor the original capitalization. ipex-LLM should be IPEX-LLM. See: 
https://ipex-llm.readthedocs.io/en/latest/doc/LLM/index.html
Pls update other cases as well.

**Review by carsonwang — APPROVED** [review]
Thanks!

**Comment by carsonwang** [comment]
@yutianchen666 , this has been merged. But need to do some cleanups. Can you please submit a new PR to delete the following files and folders. Please check if there are a few more.
https://github.com/intel/llm-on-ray/tree/main/llm_on_ray/inference/models/bigdl
https://github.com/intel/llm-on-ray/blob/main/dev/docker/Dockerfile.bigdl-cpu
https://github.com/intel/llm-on-ray/blob/main/docs/serve_bigdl.md

**Comment by yutianchen666** [comment]
> @yutianchen666 , this has been merged. But need to do some cleanups. Can you please submit a new PR to delete the following files and folders. Please check if there are a few more. https://github.com/intel/llm-on-ray/tree/main/llm_on_ray/inference/models/bigdl https://github.com/intel/llm-on-ray/blob/main/dev/docker/Dockerfile.bigdl-cpu https://github.com/intel/llm-on-ray/blob/main/docs/serve_bigdl.md

ok

**Pull request merged**